server/portal/apps/auth/views.py

```diff
@@ -32,30 +32,35 @@ def _get_auth_state():
     return secrets.token_hex(24)
 
 
-def agave_oauth(request):
-    """First step for agave OAuth workflow.
+def tapis_oauth(request):
+    """First step for Tapis OAuth workflow.
     """
-    tenant_base_url = getattr(settings, 'AGAVE_TENANT_BASEURL')
-    client_key = getattr(settings, 'AGAVE_CLIENT_KEY')
-
     session = request.session
     session['auth_state'] = _get_auth_state()
     next_page = request.GET.get('next')
     if next_page:
         session['next'] = next_page
 
-    redirect_uri = 'https://{}{}'.format(request.get_host(),
-                                         reverse('portal_auth:agave_oauth_callback'))
-    logger.debug('redirect_uri %s', redirect_uri)
+    if request.is_secure():
+        protocol = 'https'
+    else:
+        protocol = 'http'
+
+    redirect_uri = f"{protocol}://{request.get_host()}{reverse('portal_auth:tapis_oauth_callback')}"
+
+    tenant_base_url = getattr(settings, 'TAPIS_TENANT_BASEURL')
+    client_id = getattr(settings, 'TAPIS_CLIENT_ID')
+
     METRICS.debug("user:{} starting oauth redirect login".format(request.user.username))
```

Comment:
Can we start moving these `format` calls to f-strings while we're rewriting these views, to keep the string formatting consistent?